### ISSUE TITLE
[Bug]: UEN number require number only. UEN contains alphanumeric

### ISSUE THREAD

**Issue by wheel1992** [winstxnhdw/sgqr#77]
### What happened?

My company UEN is `2018...NABC`, but an error was thrown 

```
Type '"2018...NABC"' is not assignable to type '"0" | "+10" | "+70" | "+930" | "+3550" | "+2130" | "+3760" | "+2440" | "+540" | "+3740" | "+2970" | "+610" | "+6720" | "+430" | "+9940" | "+9730" | "+8800" | "+3750" | "+320" | "+5010" | ... 197 more ... | "+9700"'.
```

Implementation
```
await sgqr.generate_code({
    number: '201830304NBBC',
    number_type: 'UEN',
    ...
  });
```

How should we allow alphanumeric?

### Acknowledgement

- [X] I have done my due diligence to check for similar issues

**Comment by wheel1992** [comment]
@winstxnhdw

**Comment by winstxnhdw** [comment]
Oops. I made an assumption that UEN can only be numeric. This is an easy fix. I’ll push a hotfix later.

**Comment by winstxnhdw** [comment]
Fixed in `v0.8.1`
